config.yaml

```diff
@@ -100,3 +100,28 @@ options:
         beyond the pool size when the pool is exhausted.
     default: 10
     type: int
+  google-client-id:
+    description: |
+      Client id from Google Oauth setup
+    type: string
+  google-client-secret:
+    description: |
+      Client password from Google Oauth setup
+    type: string
+  oauth-domain:
+    description: |
+      The domain for which to allow google authentication ie. canonical.com
```

Comment:
`google` to `Google` and `ie.` to `i.e.` may be more appropriate but it is not important.

Reply:
Thanks will update.